README.md

```diff
@@ -135,14 +136,14 @@ The only thing you need to install is Docker. All of the other dev dependencies
     <summary>More information</summary>
 
     1. You will see a "BUILD COMPLETE" for each module and no error messages in the terminal output.
-	2. Open `config/deployment.json` file, you will see the module image placeholders expanded correctly.
+	2. Open `config/deployment.amd64.json` file, you will see the module image placeholders expanded correctly.
```

Comment:
Shall we show the default platform configuration in .env?

Reply:
Added a description of the platform being built without `--platform` parameter.